test/models/local_restriction_test.rb

```diff
@@ -1,125 +1,175 @@
 require "test_helper"
 
 describe LocalRestriction do
-  let(:restriction) { described_class.new("E08000123") }
-
-  before do
-    LocalRestriction.any_instance.stubs(:file_name).returns("test/fixtures/local-restrictions.yaml")
+  describe ".all" do
+    it "contains an array of LocalRestriction models" do
+      assert(described_class.all.all? { |item| item.is_a?(described_class) })
+    end
   end
 
-  it "returns the area name" do
-    assert_equal "Tatooine", restriction.area_name
-  end
+  describe ".find" do
+    it "can find a LocalRestriction by gss_code" do
```

Comment:
Ah yes, that's a good point. It seems pretty meaningless to assert the type since we already assert that `.all` returns items matching the type. I've updated this to match the `gss_code`.